index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <div id="header">
+     
+    <img src="images\original_trombones.png" alt="company-logo" id="header-img">
+     <nav id="nav-bar">
+
+     <a href="#" class="nav-link">features</a>
+     <a href="#" class="nav-link">how it works</a>
+     <a href="#" class="nav-link">pricing</a>
+     </nav>
+
+
+   </div>
+
+   <form action=" https://www.freecodecamp.com/email-submit" id="form">
+    <p>
+      Handcrafted, home-made masterpieces
+    </p>
+    <input type="email" name="email" placeholder="Enter your email address" id="email">
+    <input type="submit" value="Get started" id="submit">
```

Comment:
- [ ] Let's kindly put a required attribute to the input fields. This ensures that the form validation is activated on submit, and prevents your users from skipping crucial information 👍